spec/builtin_contracts_spec.rb

```diff
@@ -3,296 +3,303 @@
     @o = GenericExample.new
   end
 
+  def fails(&some)
+    expect { some.call }.to raise_error(ContractError)
+  end
+  def passes(&some)
```

Comment:
Can you add a blank line before the `def passes(&some)` ? Currently this PR is failing because rubocop is complaining about the missing blank line: https://travis-ci.org/egonSchiele/contracts.ruby/jobs/185597019